openfl-tutorials/interactive_api/PyTorch_MVTec_PatchSVDD/workspace/requirements.txt

```diff
@@ -0,0 +1,143 @@
+absl-py==0.13.0
```

Comment:
Why do we need this file? First of all, it looks like it contains too many requirements. Second, it's not used inside notebook. You need to either have a requirements.txt file and use it inside notebook, or directly list all requirements by name inside notebook